src/search.rs

```diff
@@ -0,0 +1,156 @@
+//! BlindIndexSearch - Search SDK for working with Blind Indexes.
+//!
+//! This is a technique that allows you to hide the terms that have been indexed. This particular implementation uses tri-grams, which
+//! are salted and hashed to produce the list of tokens.
+//!
+//! ## [BlindIndexSearch](BlindIndexSearch.html)
+//!
+//! The BlindIndexSearch gives the ability to generate queries as well as create the search entries to store.
+//!
+
+use crate::document::advanced::DocumentEncryptUnmanagedResult;
+use crate::document::advanced::*;
+use crate::document::DocumentEncryptOpts;
+use crate::internal::{take_lock, IronOxideErr};
+use crate::Result;
+use crate::{GroupId, IronOxide};
+use async_trait::async_trait;
+use rand::{self, RngCore};
+use std::collections::HashSet;
+use std::convert::{TryFrom, TryInto};
+use std::ops::DerefMut;
+
+use ironcore_search_helpers::generate_hashes_for_string;
+
+///The required length of the salt.
+const REQUIRED_LEN: usize = 32;
+
+///The result of creating a new index as well as initializing a BlindIndexSearch.
+///If you only want to create the index, see create_index.
+pub struct BlindIndexCreateResult {
+    pub encrypted_salt: EncryptedBlindIndexSalt,
+    pub sdk: BlindIndexSearch,
+}
```

Comment:
This looks to be unused now, which matches up with what we discussed in the meeting.

Reply:
Removed